### DISCUSSION TITLE
스크린샷 기능 아이폰에서 save와 share가 미작동하는 오류 ㅠ.ㅠ

### DISCUSSION THREAD

**Discussion by kimyang-sun** [naverz/zepeto-studio-kor#1247] [5. Q&A]
안녕하세요!
만든맵을 테스트 링크를 통해서 핸드폰으로 테스트중인데
스크린샷 기능을 예제에서 받아다 적용하구 사진찍고 저장, 공유, 피드에 올리기 등등 갤럭시에선 다 잘되는 상태입니다.
하지만  아이폰에서는 사진 찍은 후에 피드에 올리기는 잘되는데
저장, 공유하기가 되질 않습니다.

저장같은 경우는 클릭했을때 저장되었다고는 뜨는데 핸드폰 갤러리에 이미지가 생성되지 않고
공유하기는 무반응 상태입니다.

혹시 아이폰에선 스크린샷이 잘 안되는 버그가 있는걸까요? (예제에 있는 스크린샷을 가져왔음)

테스트 기기는 아이폰 12미니입니다.

**Comment by kimyang-sun** [comment]
앗.. 아이폰에선 앱권한 허용 팝업이 안떠서 앱 권한을 따로 허용으로 설정해주니까 잘 되네요 ㅎㅎ..
해결했습니다!

**Reply by zlmrzlo** [reply]
나중에 꿀팁으로 사용될 것 같아요! 확인해주셔서 감사합니다.
혹시 앱 권한 중에 어떤 속성을 허용으로 해주셨는지 알 수 있을까요?

**Reply by kimyang-sun** [reply]
@zlmrzlo 사진 사용, 저장공간 사용 권한 이런부분을 허용으로 해줬습니다 ㅎㅎ
